BE/src/main/java/baseball/controller/GameController.java

```diff
@@ -0,0 +1,35 @@
+package baseball.controller;
+
+import baseball.service.GameService;
+import baseball.service.dto.GameDTO;
+import baseball.service.dto.GameScoreDTO;
+import baseball.service.dto.RequestScoreDTO;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+
+@RestController
+@RequestMapping("/games")
+public class GameController {
+
+    private GameService gameService;
+
+    public GameController(GameService gameService) {
+        this.gameService = gameService;
+    }
+
+    @GetMapping
+    public List<GameDTO> showGames() {
+        return gameService.getGameDTOList();
+    }
+
+    @PostMapping("/{teamId}/score")
+    public void saveScore(@PathVariable Long teamId, @RequestBody RequestScoreDTO scoreDTO) {
```

Comment:
```suggestion
    public void saveScore(@PathVariable Long teamId, @RequestBody ScoreRequest scoreRequest) {
```
저는 가급적이면 요청과 응답에 대한 DTO를 네이밍 할 때에는 위의 네이밍 전략을 사용하고 있습니다.
DTO라는 불필요한 용어를 제거할 수 있다면 제거하는게 좋다고 생각해요.

Reply:
감사합니다! RequestxxxDTO 형태로 작성한 클래스명들 전부다 변경했습니다.